app/views/content/terms_candidate.md

```diff
@@ -1,40 +1,53 @@
-Thank you for using Apply for teacher training, a service run by the Department for Education.
+Terms are a mixture of rules and things to know, like what we do with your data. Applying for teacher training here means you agree to our terms.
 
-Using Apply for teacher training means you agree to these terms of use.
+If you don’t follow them your application might be withdrawn.
```

Comment:
We should avoid negative contractions for consistency.

Change 'don't' to 'do not'